addons/crm/static/src/views/crm_form/crm_pls_tooltip_button.js

```diff
@@ -0,0 +1,79 @@
+import { Component, status } from "@odoo/owl";
+import { standardWidgetProps } from "@web/views/widgets/standard_widget_props";
+
+import { localization } from "@web/core/l10n/localization";
+import { registry } from '@web/core/registry';
+import { usePopover } from "@web/core/popover/popover_hook";
+import { useService } from "@web/core/utils/hooks";
+
+export class CrmPlsTooltip extends Component {
+    static props = {
+        close: { optional: true, type: Function },
+        dashArrayVals: {type: String},
+        low3Data: { optional: true, type: Object },
+        probability: { type: Number },
+        teamName: { optional: true, type: String },
+        top3Data: { optional: true, type: Object },
+    };
+    static template = "crm.PlsTooltip";
+}
+
+
+export class CrmPlsTooltipButton extends Component {
+    static template = "crm.PlsTooltipButton";
+    static props = {...standardWidgetProps};
+
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+        this.popover = usePopover(CrmPlsTooltip, {
+            popoverClass: 'mt-2 me-2',
+            position: "bottom-start",
+        });
+    }
+
+    async onClickPlsTooltipButton(ev) {
+        const tooltipButtonEl = ev.currentTarget;
+        if (this.popover.isOpen) {
+            this.popover.close();
+        } else {
+            // Apply pending changes. They may change probability
+            await this.props.record.save();
+            if (status(this) === "destroyed") {
+                return;
+            }
```

Comment:
Not sure if it's because of this and if we can do something, but sometimes I need two clicks on the IA button to open the popover. Happens only when we have a `~ X%` value that is updated it seems

Reply:
Yes, as tooltip indicates, same icon but two actions : 1) align to auto proba 2) open tooltip